src/styles/utils/main__row.scss

```
@@ -0,0 +1,7 @@
+.main__row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-grow: 1;
+  margin-top: 49px;
+}
```

Also, rename the file by block name, not its element.

```suggestion
.main {
  &__row {
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    flex-grow: 1;
    margin-top: 49px;
  }
}
```